packages/webview_flutter/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: webview_flutter_tizen
+description: Tizen implementation of the webview plugin
+homepage: https://github.com/flutter-tizen/plugins
+version: 1.0.0
```

Comment:
Something like `0.1.0` would be better as this is a beta release.

Reply:
Done.